packages/shared/src/lib/pieces/utils.ts

```diff
@@ -0,0 +1,16 @@
+type PackageInfoParams = {
+  pieceName: string;
+  pieceVersion: string;
+}
+
+export const getPackageNameForPiece = (params: PackageInfoParams): string => {
```

Comment:
I assume this is the alias inside activepieces right? can you we rename it to getPackageInternalNameForPiece / or getPackageAliasForPiece?

Reply:
fixed in 4b91624